doc/source/whatsnew/v0.24.0.rst

```diff
@@ -678,9 +680,9 @@ is now an extension array (:issue:`21978`, :issue:`19056`, :issue:`22835`).
 To conform to this interface and for consistency with the rest of pandas, some API breaking
 changes were made:
 
-- ``SparseArray`` is no longer a subclass of :class:`numpy.ndarray`. To convert a SparseArray to a NumPy array, use :meth:`numpy.asarray`.
+- ``SparseArray`` is no longer a subclass of :class:`numpy.ndarray`. To convert a ``SparseArray`` to a NumPy array, use :func:`numpy.asarray`.
 - ``SparseArray.dtype`` and ``SparseSeries.dtype`` are now instances of :class:`SparseDtype`, rather than ``np.dtype``. Access the underlying dtype with ``SparseDtype.subtype``.
-- :meth:`numpy.asarray(sparse_array)` now returns a dense array with all the values, not just the non-fill-value values (:issue:`14167`)
+- ``numpy.asarray(sparse_array)`` now returns a dense array with all the values, not just the non-fill-value values (:issue:`14167`)
```

Comment:
Did the old link to numpy.asarray not work? Or does it just need to be changed from meth to func?